elm.json

```diff
@@ -5,19 +5,20 @@
     "license": "BSD-3-Clause",
     "version": "1.1.1",
```

Comment:
`1.2.0`

Reply:
or does Elm handle that?

Reply:
Right, I suppose I could run `elm bump`. I didn't bother to increase the version so far because you need to publish it manually after merging to master anyways, right?

Reply:
I think so, has been a while